pkg/k8s/labels.go

```diff
@@ -0,0 +1,55 @@
+package k8s
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+const (
+	// LabelName is used for specifying the name of the app.
+	LabelName = "app.kubernetes.io/name"
+	// LabelComponent is used for specifying a specific component of the app.
+	LabelComponent = "app.kubernetes.io/component"
+	// LabelPartOf is used for specifying the name of a higher level app it is part of.
+	LabelPartOf = "app.kubernetes.io/part-of"
+
+	// AppName is the name of the app.
+	AppName = "traefik-mesh"
+
+	// ComponentProxy is component of type proxy.
+	ComponentProxy = "proxy"
+	// ComponentShadowService is component of type shadow-service.
+	ComponentShadowService = "shadow-service"
+)
+
+// ShadowServiceLabels are the labels of a shadow service.
+var ShadowServiceLabels = map[string]string{
```

Comment:
WDYT of using a function here to avoid a mutable shared var and to be consistent with the selector functions. Something like:

```suggestion
func ShadowServiceLabels() map[string]string
```